src/basics.md

```diff
@@ -748,6 +749,63 @@ fn run() -> Result<()> {
 
 ```
 
+[ex-sha-digest]: #ex-sha-digest
+<a name="ex-sha-digest"></a>
+## Calculate the SHA-256 digest of a file
+
+[![ring-badge]][ring] [![data-encoding-badge]][data-encoding] [![cat-filesystem-badge]][cat-filesystem]
+
+Writes some data to a file, then calculates the SHA-256 digest of
```

Comment:
I would probably changed "digest" to linked  [`Digest`](https://docs.rs/ring/*/ring/digest/struct.Digest.html).